packages/utils/src/lib/reports/__snapshots__/report-diff-monorepo-with-portal.md

```diff
@@ -0,0 +1,48 @@
+# Code PushUp
+
+🥳 Code PushUp report has **improved** – compared target commit 0123456789abcdef0123456789abcdef01234567 with source commit abcdef0123456789abcdef0123456789abcdef01.
+
+## 💼 Project `frontoffice` – improved 🥳
```

Comment:
I'm not sure if we should only show the project name and a icon to have more space for longer project names...

Reply:
I'm trying to figure out the right balance between compactness and being easily understandable to an uninitiated user. I don't think we should rely on just an emoji to convey information (see [icons need a text label](https://www.nngroup.com/articles/icon-usability/#:~:text=wasted%20feature.-,Icons%20Need%20a%20Text%20Label,%2C%20but%20it%20also%20fails%20to%20translate%20well%20on%20touch%20devices.,-The%20website%20Usability) design principle).

Here are some alternative layouts I came up with:

---

_Alternative 1: extra nesting_ :arrow_down: 

---

## 💼 Projects

### `frontoffice` – improved 🥳

...

### `backoffice` – regressed 😟

...

---

_Alternative 2: status as first paragraph_ :arrow_down: 

---

## 💼 Project `frontoffice`

🥳 Code PushUp report has **improved**.

...

## 💼 Project `backoffice`

😟 Code PushUp report has **regressed**.

...

Reply:
Went with 2nd alternative in 11eb84f13ff9af3d4c646bfba0a68ee69eb9c2d1.